{
    "dataType": "CVE_RECORD",
    "cveMetadata": {
        "state": "PUBLISHED",
        "cveId": "CVE-2024-33662",
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "dateUpdated": "2024-10-02T04:33:04.879794",
        "dateReserved": "2024-04-25T00:00:00",
        "datePublished": "2024-10-02T00:00:00"
    },
    "containers": {
        "cna": {
            "providerMetadata": {
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre",
                "dateUpdated": "2024-10-02T04:33:04.879794"
            },
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Portainer before 2.20.2 improperly uses an encryption algorithm in the AesEncrypt function."
                }
            ],
            "affected": [
                {
                    "vendor": "n/a",
                    "product": "n/a",
                    "versions": [
                        {
                            "version": "n/a",
                            "status": "affected"
                        }
                    ]
                }
            ],
            "references": [
                {
                    "url": "https://www.portainer.io"
                },
                {
                    "url": "https://github.com/portainer/portainer/compare/2.20.1...2.20.2"
                },
                {
                    "url": "https://github.com/portainer/portainer/issues/11737"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "type": "text",
                            "lang": "en",
                            "description": "n/a"
                        }
                    ]
                }
            ]
        }
    },
    "dataVersion": "5.1"
}